style.css

```diff
@@ -96,9 +104,41 @@ body{
 }
 .category-list__list-item:hover{
     cursor: pointer;
-    background-color: #8080801c
+    background-color: #8080801c;
 }
+#more li{
+    border: 0.002rem solid #00000036;
+    border-top: none;
+    padding: 1.5rem;
+    display: none;
+}
+#dropdown {
+    position: absolute;
+    z-index: 1;
+    top: 14.2rem;
+    padding: 0rem;
+    display: none;
+    background: white;
+    border: 0.002rem solid #00000036;
+    border-top: none;
+    right: 10rem;
```

Comment:
you should write top and right together, In short related properties should be placed together, it gives a better readability